ui/pages/bridge/types.ts

```diff
@@ -1,3 +1,25 @@
+import { BigNumber } from 'bignumber.js';
+
+export type L1GasFees = {
+  l1GasFeesInHexWei?: string; // l1 fees for approval and trade in hex wei, appended by controller
+};
+
+// Values derived from the quote response
+export type QuoteMetadata = {
+  totalNetworkFee: { raw: BigNumber; fiat: BigNumber | null }; // gasFees + relayerFees
```

Comment:
`raw` means the atomic amount right?

Reply:
Yea, I'll rename this to amount since that seems to be the convention in other utils (i.e. `calcTokenAmount`)